src/sizzle.js

```diff
@@ -503,10 +503,9 @@ setDocument = Sizzle.setDocument = function( node ) {
 	docElem = document.documentElement;
 	documentIsHTML = !isXML( document );
 
-	// Support: IE 9 - 11
+	// Support: IE 9 - Edge
```

Comment:
This suggests Edge is another IE, I'd write:
```js
// Support: IE 9 - 11, Edge
```
We'll probably add some kind of versioning to Edge later (maybe by build numbers, see the discussion [from this comment forward](https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-117856894)) but until that's decided it doesn't matter too much.